silnlp/nmt/experiment.py

```diff
@@ -3,6 +3,7 @@
 import os
 from dataclasses import dataclass
 from silnlp.common.environment import SIL_NLP_ENV
+from silnlp.nmt.test import test
```

Comment:
Can you make this a relative import so that it is consistent with the rest of the codebase?